### PR TITLE
resource/alicloud_ddoscoo_domain_resource: Removed the `ForceNew` for field `proxy_types` and `rs_type`, supports modifying them online

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

File: alicloud/resource_alicloud_ddoscoo_domain_resource_test.go [hunk 1]
```diff
@@ -48,8 +47,8 @@ func TestAccAlicloudDdoscooDomainResource_basic(t *testing.T) {
 					"domain": name,
 					"proxy_types": []map[string]interface{}{
 						{
-							"proxy_ports": []string{"443"},
-							"proxy_type":  "https",
+							"proxy_ports": []string{"80"},
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_ddoscoo_domain_resource_test.go:50]: list 类型的，多测几个值

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: alicloud/resource_alicloud_ddoscoo_domain_resource_test.go (hunk 1 above)
Comment (MrWolong) [reply alicloud/resource_alicloud_ddoscoo_domain_resource_test.go]: 已修改

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: alicloud/resource_alicloud_ddoscoo_domain_resource_test.go [hunk 2]
```diff
@@ -93,11 +100,13 @@ func TestAccAlicloudDdoscooDomainResource_basic(t *testing.T) {
 			},
 			{
 				Config: testAccConfig(map[string]interface{}{
-					"real_servers": []string{"177.167.32.11", "177.167.32.12"},
+					"real_servers": []string{"qq.com"},
```
[the remark this answers is not in the document]
Comment (MrWolong) [reply alicloud/resource_alicloud_ddoscoo_domain_resource_test.go]: 已修复

**Review by xiaozhu36 — APPROVED** [review]

File: alicloud/resource_alicloud_ddoscoo_domain_resource_test.go [hunk 3]
```diff
@@ -93,28 +100,37 @@ func TestAccAlicloudDdoscooDomainResource_basic(t *testing.T) {
 			},
 			{
 				Config: testAccConfig(map[string]interface{}{
-					"real_servers": []string{"177.167.32.11", "177.167.32.12"},
+					"real_servers": []string{"qq.com", "baidu.com", "alibaba.com"},
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_ddoscoo_domain_resource_test.go:103]: 这个还是没改啊

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: alicloud/resource_alicloud_ddoscoo_domain_resource_test.go (hunk 3 above)
Comment (MrWolong) [reply alicloud/resource_alicloud_ddoscoo_domain_resource_test.go]: 已修复

**Review by xiaozhu36 — APPROVED** [review]

**Pull request merged**